### DISCUSSION TITLE
SoD Servers List

### DISCUSSION THREAD

**Discussion by srPeanuts** [simulationcraft/simc#8500] [Q&A] [answered]
### Hello team of SimC,

My question to you is:

 - Will it be possible to run simulations on SoD characters?

> At the moment with version [ _simc-1020.01.e625045-win64_ ] the app shows no SoD servers on the import tab

Thank you for your time

**Comment by vituscze** [comment, marked as answer]
Most likely no, as there are no devs working on classic.
